packages/std/src/results/cosmos_msg.rs

```diff
@@ -46,6 +46,7 @@ pub enum CosmosMsg<T = Empty> {
     Wasm(WasmMsg),
     #[cfg(feature = "stargate")]
     Gov(GovMsg),
+    LastMsgMark(T),
```

Comment:
Can we make it prettier, i.e. so dev won't need to pass a boilerplate `Empty {}` as `T`?

Reply:
Yeah that alright, just asking out of curiosity. It's a matter of optimizing dev-x at this point.